releases/fluentd-splunk-objects.yaml

```diff
@@ -0,0 +1,54 @@
+releases:
+
+#######################################################################################
+## Fluentd to Splunk logs                                                            ##
+## Forward logs to Splunk with fluentd                                               ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/splunk/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-kubernetes-logging/values.yaml
+#
+
+- name: "fluentd-splunk-objects"
```

Comment:
why not just call it `fluentd-splunk` to keep it short & sweet?

Reply:
Need to differentiate it from fluentd-splunk-logs

Reply:
oh, sounds like it has nothing to do with `fluentd` any more?

Reply:
Splunk [uses fluentd](https://github.com/splunk/splunk-connect-for-kubernetes/tree/master/helm-chart/splunk-kubernetes-objects#splunk-collect-for-kubernetes-objects)